src/Common/src/Interop/Windows/BCrypt/Interop.BCryptExportKey.cs

```diff
@@ -12,10 +12,8 @@ internal partial class Interop
 {
     internal partial class BCrypt
     {
-#if !uap
```

Comment:
It doesn't compile without it. The API is incorrectly marked as not available on UAP.

Reply:
Actually this is why I started this PR in the first place. I was afraid more of the APIs were incorrectly marked and not spotted because of the missing defines. Gladly this was the only one.